docs/source/markdown/podman-container-restore.1.md

```diff
@@ -93,6 +93,10 @@ be used once and the restored *container* will have another IP address. This als
 that **--name, -n** cannot be used in combination with **--tcp-established**.\
 *IMPORTANT: This OPTION is only available in combination with **--import, -i**.*
 
+#### **--noout**
+
+Do not print output ids
```

Comment:
Maybe "Do not print any output on success"?

Reply:
I think `Do not print any IDs in the output.`